src/Catty/Helpers/Util/Util.swift

```diff
@@ -50,6 +50,16 @@ func synchronized(lock: AnyObject, closure: () -> Void) {
         return appBuildVersion
     }
 
+    class func alert(withText text: String?) {
```

Comment:
Please make the parameter non-optional since an alert without text would not make sense. Also, I would suggest to change the Swift method signature to `alert(text: String)` (ObjC should remain `alertWithText:`).

Reply:
I have changed the method signatures according to your suggestion. But now I have the problem that all the (already existing) swift files expect the the signature to be `Util.alert(withText: String)` and so the compiler shows an error. Now I am now sure if understood you wrong, but do you want me to also change those ~20 files also to `Util.alert(text: String)`?

Reply:
Yes, please change it. However make sure, that this only affects Swift files, there should be no changes in the Objective C files.

Reply:
I changed all the Swift files, but now the ObjC files are also showing compiler errors. As I see it (and according to my web research), the signature needs to be `Util.alert(withText: String)`, in that case neither Swift nor ObjC files would need to be changed. Or I cloud add an `@objc` attribute in Swift, so that the signatures are different?

Reply:
Yes, there should be no changes in the Objective C file, the Objective C signature is perfectly fine and confirms with our standard. You will need to add `@objc(alertWithText:) `for example.

Reply:
Attributes have been added, works fine 👌🏼